carmin.yaml

```diff
@@ -51,6 +51,23 @@ paths:
                 $ref: '#/components/schemas/Authentication'
         default:
             $ref: '#/components/responses/Error'
+  /registration:
```

Comment:
@glatard What are your thoughts on this description? Any improvement or adjustment to do?

Reply:
A verb (/register) would be more consistent with the rest of the API than a nount (/registration). It could also be put under /users to enable the addition of other user management methods, in the future. Otherwise, it looks great!